src/cmd-remote-build-container

```diff
@@ -14,14 +15,14 @@ logging.basicConfig(level=logging.INFO,
                     format="%(asctime)s %(levelname)s - %(message)s")
 
 
-def build_container_image(labels, gitURL, gitRef, repo, tag):
+def build_container_image(labels, gitDir, repo, tag):
     '''
     Build the image  using podman remote and push to the registry
     @param repo str registry repository
     @param tag  str image tag
     @param time str expiration time for Quay images
```

Comment:
Seems we can remove the doc for @param time here

Reply:
will fix.